### PR TITLE
{data}[intel/2016b] HTSeq v0.6.1p1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2677.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/390c75c4e8eb7bef51d307ebe6bb39ec for a full test report.

File: easybuild/easyconfigs/h/HTSeq/HTSeq-0.6.1p1-intel-2016b-Python-2.7.12.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('Python', '2.7.12'),
+    ('matplotlib', '1.5.1', versionsuffix),
+]
+
+options = {'modulename': '%(name)s'}
```
Comment (wpoely86) [inline easybuild/easyconfigs/h/HTSeq/HTSeq-0.6.1p1-intel-2016b-Python-2.7.12.eb]: isn't this the default?

Comment (boegel) [inline easybuild/easyconfigs/h/HTSeq/HTSeq-0.6.1p1-intel-2016b-Python-2.7.12.eb]: no, the default is `%(namelower)s`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2040.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b5f65cd916243d04138021a153249edc for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**